From 01b8cc43fec92c7cc217b03f17d50d0751217f81 Mon Sep 17 00:00:00 2001
From: Arie Brown <abrown@habana.ai>
Date: Thu, 10 Sep 2020 17:20:06 +0300
Subject: [PATCH] [SW-19168] cmd: i2c: remove prints

remove prints which flooded the uart. with this prints flood, after
few minutes we could not find valuable prints on the journal log over
the bmc.

Change-Id: If858e13d8d1bef15faa8de31dd01ba6a7a7176cc
Signed-off-by: Arie Brown <abrown@habana.ai>
---

diff --git a/common/module_i2c_slave.c b/common/module_i2c_slave.c
index aed5075..f560955 100644
--- a/common/module_i2c_slave.c
+++ b/common/module_i2c_slave.c
@@ -144,10 +144,8 @@
 	stat = regmap_readl(mmPSOC_I2C_S_IC_INTR_STAT);
 
 	if ((stat & PSOC_I2C_S_IC_INTR_STAT_R_RD_REQ_MASK) ||
-	    (stat & PSOC_I2C_S_IC_INTR_STAT_R_RX_FULL_MASK)) {
-		preboot_puts("I2C SLAVE: read request received\n");
+	    (stat & PSOC_I2C_S_IC_INTR_STAT_R_RX_FULL_MASK))
 		res = FOUND_FAST_RETURN;
-	}
 
 	return res;
 }
@@ -222,7 +220,6 @@
 		ret = i2cslave_handle_read_req(&data);
 		data.read_bytes_count++;
 		regmap_writel(ret, mmPSOC_I2C_S_IC_DATA_CMD);
-		preboot_puts("I2C SLAVE: read request processed\n");
 	}
 }
 
